app/views/teams/inviting.html.erb

```diff
@@ -1,6 +1,11 @@
+<p id="notice"><%= notice %></p>
+
 <p>
-    <strong>Users invited to join Team <%= @team.name %></strong>
+    <strong>Users invited to join <%= @team.name %></strong>
 </p>
+<% if @invitations.nil? || @invitations.empty? %>
```

Comment:
I think it's enough to check `@invitations.empty?` because `invitations` is an array.